pkgs/development/tools/misc/babeltrace/default.nix

```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
```

Comment:
After the 1.5.8 -> 1.5.11 commit, or even before it, it'd be nice to format with `nixfmt` the expression, and also switch to `finalAttrs:` pattern.

Reply:
I reformatted it as last commit (no time for resolving conflicts right now :-). 

Also, is there any advantage of adding the `finalAttrs` parameter if it is not used in the actual Nix expression? I don't think so and I believe that it makes evaluation a bit slower.

Reply:
Here we could replace `rec` and `version` with `finalAttrs`, but it doesn't help much, because we would need to override the whole `src` due to `hash` change. Or am I missing something?